jsk_panda_robot/panda_eus/CMakeLists.txt

```diff
@@ -34,14 +34,30 @@ if(franka_description_FOUND
     WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/models
     DEPENDS ${PROJECT_SOURCE_DIR}/models/dual_panda.urdf.xacro)
 
-  add_custom_target(generate_panda_lisp ALL DEPENDS ${PROJECT_SOURCE_DIR}/models/dual_panda.l)
+  add_custom_target(generate_dual_panda_lisp ALL DEPENDS ${PROJECT_SOURCE_DIR}/models/dual_panda.l)
 else()
   message(WARNING "Dependency is not met, so skip generating dual_panda.l")
   message(WARNING "franka_description version: ${franka_description_VERSION}, must be >= ${_franka_description_min_ver}")
   message(WARNING "xacro version: ${xacro_VERSION}, must be >= ${_xacro_min_ver}")
 endif()
 
 
+###
+### panda.l generation
+###
+add_custom_command(OUTPUT ${PROJECT_SOURCE_DIR}/models/panda.l
```

Comment:
Please move `add_custom_command` and `add_custom_target` for `panda.l` into the same `if` block as `dual_panda.l` generation, because you cannot generate `panda.l` outside that `if` block.
Please read https://github.com/jsk-ros-pkg/jsk_robot/blob/93919243019ae06be43874022062abbeee00349e/jsk_panda_robot/panda_eus/CMakeLists.txt#L21-L27
Maybe you have to change some messages:
`dual_panda.urdf.xacro assumes` -> `panda.urdf.xacro and dual_panda.urdf.xacro assume` in
https://github.com/jsk-ros-pkg/jsk_robot/blob/93919243019ae06be43874022062abbeee00349e/jsk_panda_robot/panda_eus/CMakeLists.txt#L22
`dual_panda.l` -> `panda.l and dual_panda.l` in
https://github.com/jsk-ros-pkg/jsk_robot/blob/93919243019ae06be43874022062abbeee00349e/jsk_panda_robot/panda_eus/CMakeLists.txt#L39

Reply:
move the code for `panda.l` into `if` block and modify messages.
[7a093af](https://github.com/jsk-ros-pkg/jsk_robot/pull/1799/commits/7a093afe919bc2038fde6d3611f7cf2fba24eed9)